admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/server/service/persistence/module/FieldManager.java

```diff
@@ -48,37 +49,57 @@
 import javax.persistence.metamodel.EntityType;
 
 /**
- * 
+ *
  * @author jfischer
  *
  */
 public class FieldManager {
-    
+
     private static final Log LOG = LogFactory.getLog(FieldManager.class);
 
     public static final String MAPFIELDSEPARATOR = "---";
 
     protected EntityConfiguration entityConfiguration;
     protected EntityManager entityManager;
     protected List<SortableValue> middleFields = new ArrayList<SortableValue>(5);
+    protected Set<Class> classes = new HashSet<>();
```

Comment:
Maybe rename this variable to `managedEntityClasses` to provide better traceability into what this is doing?